### PR TITLE
Update README.md fix  PSI Quick Start with v2 API  data path

### PR DESCRIPTION
**Pull request by GreyJeremyji**
Update README.md fix  PSI Quick Start with v2 API  data path

### CODE REVIEW

**Review by 6fj — APPROVED** [review]
lgtm

**Comment by 6fj** [comment]
hi @GreyJeremyji 

Please sign CLA according to instructions above, thanks!

**Comment by GreyJeremyji** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/secretflow/.github/blob/main/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> I have read the CLA Document and I hereby sign the CLA
> 
> You can retrigger this bot by commenting **recheck** in this Pull Request

I have read the CLA Document and I hereby sign the CLA

**Comment by 6fj** [comment]
recheck

**Comment by 6fj** [comment]
Hi @GreyJeremyji  You don't need to quote bot comments.

File: README.md
Comment (GreyJeremyji) [inline README.md:1]: I have read the CLA Document and I hereby sign the CLA

Comment (6fj) [reply README.md:1]: Please comment in Conservation tag instead

**Comment by GreyJeremyji** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by 6fj** [comment]
recheck

**Pull request merged**